Regression/WarpX-tests.ini

```diff
@@ -670,7 +671,7 @@ particle_tolerance = 1.e-12
 [Python_Langmuir]
 buildDir = .
 inputFile = Examples/Tests/Langmuir/PICMI_inputs_langmuir_rt.py
-runtime_params =
+runtime_params = diag1.file_prefix=Python_Langmuir_plt
```

Comment:
Note that runtime_params will be ignored for the Python test cases. In my PR, I'll make the appropriate changes to the input files, for example so the data is written out to the plot files with the same names as before.

Reply:
Good point, thanks, then I will delete these lines as they are useless.